addons/website/static/src/snippets/s_countdown/options.js

```diff
@@ -12,6 +12,11 @@ options.registry.countdown = options.Class.extend({
         'click .toggle-edit-message': '_onToggleEndMessageClick',
     }),
 
+    cleanForSave: async function () {
+        this.$target.find('.s_countdown_canvas_wrapper').removeClass("s_countdown_none");
+        this.$target.find('.s_countdown_end_message').removeClass("s_countdown_enable_preview");
+        this._super(...arguments);
+    },
```

Comment:
- Missing blank line before options section, missing documentation markers
- No need of calling super in cleanForSave, it's meant to be forever abstract in the parent definition